packages/tools/generator-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/generator-widget",
-  "version": "8.12.1",
+  "version": "8.12.2",
```

Comment:
8.12.1 is not published yet because of 
```
i noticed that the release for generator was not successful
1:14
because you cannot release more than 1 tag with same master
```